### DISCUSSION TITLE
patchClientRoutes({ routes }) 文档示例问题

### DISCUSSION THREAD

**Discussion by laterinc** [umijs/umi#8653] [Ideas]
根据文档[patchClientRoutes({ routes })](https://umijs.org/docs/api/runtime-config#patchclientroutes-routes-)的示例测试提示：
<img width="866" alt="image" src="https://user-images.githubusercontent.com/16649952/180117164-82df586f-461c-4def-bb5f-c9619da75f12.png">
修改成如下正常加载：
`routes.unshift({
        path: '/foo',
        element: <div>111</div>
 })`

**Comment by fz6m** [comment]
问题的复现给个 demo 看看。

**Reply by laterinc** [reply]
问题复现demo ： https://github.com/artisanship/umi-demo

**Reply by laterinc** [reply]
访问新添加的 "/foo" 路由才会出现

**Reply by fz6m** [reply]
谢谢反馈，正在修正 #8660
